docs/mine-hnt/build-a-gateway/build-a-gateway.mdx

```diff
@@ -0,0 +1,179 @@
+---
+id: build-a-gateway
+hide_title: true
+sidebar_label: Build a Gateway
+slug: /mine-hnt/build-a-gateway
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+## Build a Gateway
+
+**Note**: The ability to mine HNT with anything other than prebuilt Hotspots is
+currently under development following the conclusion of the DIY Alpha program.
+Please join the Helium Discord Server and the #hotspot-diy-hardware channel for
+the latest updates on the roadmap here.
+
+<img src={useBaseUrl("img/mine-hnt/mine-hnt-arch.png")} />
+
+To be a Helium Miner, there are three principle components to know about
+
+- **Packet Forwarder**: this is a utility that interacts with the radio
+  front-end and sends and receives raw radio packets with the Helium Miner
+- **Miner**: the Helium Blockchain comes into the picture here; the Miner is
+  responsible for routing packets to the appropriate Router (see our Routing
+  article) entering into microtransactions brokered via libp2p
+- **Router**: a Helium compatible LoRaWAN Network Server, basically; this
+  component is interested in receiving the packets relating to its devices and
+  handles downlink responses when appropriate
+
+When you buy [a prebuilt Hotspot](mine-hnt/prebuilt-hotspots), you are buying a
```

Comment:
This link is bad. Relative links are pretty error prone because of the nesting we're doing, it's best to use full path if possible, which need to start with `/docs`

Reply:
Good catch. Thank you :pray: